lib/cher/error.go

```diff
@@ -152,3 +152,16 @@ func Coerce(v interface{}) E {
 func (e E) Value() (driver.Value, error) {
 	return json.Marshal(e)
 }
+
+func WrapIfNotCher(err error, msg string) error {
+	if err == nil {
+		return nil
+	}
+
+	var cErr E
+	if errors.As(err, &cErr) {
+		return cErr
+	}
+
+	return errors.Wrap(err, msg)
```

Comment:
Could use the stdlib:

`fmt.Errorf("%s: %w", msg, err)`

Reply:
However wouldn't make much difference as `github.com/pkg/errors` is already heavily used in this pkg

Reply:
Yeah I'd prefer that too, this package was already included though

Reply:
And this func kinda matches the same signature as the pkrerr.Wrap func